components/layouts/DocsLayout.js

```diff
@@ -50,3 +51,7 @@ export default function DocsLayout({ children, title }) {
     </>
   );
 }
+
+DocsLayout.getLayout = function getLayout(page) {
+  return <MultiLayout>{page}</MultiLayout>;
+};
```

Comment:
Not sure why `MultiLayout` is added to the `DocsLayout`? I think it would be better to be like before but add your new component `SkipLink`